app/src/app/api/daily-bank/route.ts

```diff
@@ -86,8 +86,33 @@ export async function GET() {
         ]);
       }),
     );
+
+    // Log the reset into the DailyReset schema
+    await drizzleDB.insert(dailyReset).values({
+      id: crypto.randomUUID(), // Generate a unique ID for the reset entry
```

Comment:
This is theoretically fine, but for consistency we use the nanoid() across the site :)